slack_pull_reminder.py

```diff
@@ -7,7 +7,11 @@
 POST_URL = 'https://slack.com/api/chat.postMessage'
 
 ignore = os.environ.get('IGNORE_WORDS')
-IGNORE_WORDS = ignore.split(',') if ignore else []
+IGNORE_WORDS = [i.lower().strip() for i in ignore.split(',')] if ignore else []
+
+repos = os.environ.get('REPOS')
```

Comment:
Let's call it `repositories` and `REPOSITORIES`

Reply:
done!